regulations_and_standards.md

```diff
@@ -32,11 +32,11 @@ TEE, HSM, etc.
 [**ISO/DIS 21448 Road vehicles — Safety of the intended functionality (SOTIF),January 2021**](https://www.iso.org/standard/77490.html?browse=tc)
 This document augments ISO 26262 (Functional Safety).
 
-[**ISO/CD 24089 Road vehicles-Software update engineering, February 2021**](https://www.iso.org/standard/77796.html?browse=tc)
+[**ISO/CD 24089 Road vehicles - Software update engineering, February 2021**](https://www.iso.org/standard/77796.html?browse=tc)
 This document augments the ISO Standard 26262 (Functional Safety) and ISO/SAE Standard 21434 (Cybersecurity). It standardizes detailed process requirements for wired and wireless software updates.
 
-[**ISO TC22/SC32/WG11 ISO 21434 Road Vehicles Cybersecurity Engineering**](https://upstream.auto/blog/setting-the-standard-for-automotive-cybersecurity/) 
-A joint effort between the International Standards Organization and the Society of Automotive Engineers that brings together “more than 100 experts from more than 82 companies based in over 16 countries.” The goal is a “deep and effective global standard for automotive cybersecurity.” The initiative is divided into four main working groups that focus in on risk management; product development; production, operation, maintenance, and decommissioning; and process overview.
+[**ISO TC22/SC32/WG11 ISO 21434 Road Vehicles - Cybersecurity Engineering**](https://upstream.auto/blog/setting-the-standard-for-automotive-cybersecurity/) 
+A joint effort between the International Standards Organization and the SAE International that brings together “more than 100 experts from more than 82 companies based in over 16 countries.” The goal is a “deep and effective global standard for automotive cybersecurity.” The initiative is divided into four main working groups that focus on risk management; product development; production, operation, maintenance, and decommissioning; and process overview.
```

Comment:
Isn't it better to spell out acronyms?

Reply:
I checked this and SAE is no longer spelling out its name. SAE International is correct.